en-US/focus-ios.xliff

```diff
@@ -339,12 +359,15 @@
       <trans-unit id="Settings.toggleSendAnonymousUsageData">
         <source>Send anonymous usage data</source>
         <target>Send anonymous usage data</target>
-        <note>Label for Send Anonymous Usage Data toggle on main screen</note>
```

Comment:
Why did we lose this note? I don't see any change in the code that should cause this :-\

Reply:
See https://github.com/mozilla-mobile/focus-ios/issues/395